lib/dockleHelper.js

```diff
@@ -49,6 +52,52 @@ function getDockle() {
     });
 }
 exports.getDockle = getDockle;
+function isCisChecksEnabled() {
+    return inputHelper.addCISChecks.toLowerCase() === "true";
+}
+exports.isCisChecksEnabled = isCisChecksEnabled;
+function getSummary(dockleStatus) {
+    let summary = '';
+    switch (dockleStatus) {
+        case 0:
+            summary = 'No CIS benchmark violations were detected in the container image.';
+            break;
+        case exports.DOCKLE_EXIT_CODE:
+            summary = getCisSummary();
+            break;
+        default:
+            summary = 'An error occured while scanning the container image for CIS benchmark violations.';

```

Comment:
When will this case get hit ?

Reply:
In case some error occurs while executing dockle and it halts in the middle.